### PR TITLE
Bump publish plugin

### PR DESCRIPTION
**Pull request by jrodbx**

### CODE REVIEW

File: paparazzi/build.gradle [hunk 1]
```diff
@@ -72,8 +72,12 @@ subprojects {
       }
     }
```
Comment (gabrielittner) [inline paparazzi/build.gradle:74]: You probably want something like this here to still sign artifacts and add sonatype/central as a publishing repo
```suggestion
    mavenPublishing {
        publishToMavenCentral()
        signAllPublications()
    }
```
Alternatively you can also set to Gradle properties like described here https://github.com/vanniktech/gradle-maven-publish-plugin/blob/master/CHANGELOG.md#version-0210-2022-07-11

File: paparazzi/build.gradle [hunk 2]
```diff
@@ -72,8 +72,12 @@ subprojects {
...
 
-    mavenPublish {
-      releaseSigningEnabled = hasProperty('signingKey')
+    plugins.withId('signing') {
+      signing {
+        def signingKey = findProperty('signingKey')
+        def signingPassword = ''
+        useInMemoryPgpKeys(signingKey, signingPassword)
```
Comment (gabrielittner) [inline paparazzi/build.gradle:79]: The plugin has something built in for picking up in memory signing keys you just need to rename your `signingKey` property to `signingInMemoryKey` https://github.com/vanniktech/gradle-maven-publish-plugin#signing

File: paparazzi/build.gradle (hunk 1 above)
Comment (jrodbx) [reply paparazzi/build.gradle:74]: oh, i had done that (the 2nd approach), but looks like i forgot to add it.  that's what I get for late night PRs.  thanks for the sanity check!

**Review by oldergod — COMMENTED** [review]
lgtm

**Review by gamepro65 — APPROVED** [review]

**Review by luis-cortes — APPROVED** [review]

**Pull request merged**